net/gallery-dl/Portfile

```diff
@@ -9,7 +9,7 @@ github.tarball_from releases
 distname            gallery_dl-${github.version}
 
 categories          net
-maintainers         nomaintainer
+maintainers         {@akierig fastmail.de:akierig} \
```

Comment:
trailing backslash; this is missing "openmaintainer" (please keep those on the same line anyway